### DISCUSSION TITLE
iOS Bugly上监测到的堆栈异常，本地调试没有复现过，如果使用4.3.33版本这种情况是否会有改善？

### DISCUSSION THREAD

**Discussion by leiyuncun** [Tencent/libpag#1907] [Q&A] [answered]
![image](https://github.com/Tencent/libpag/assets/12692791/a56148db-c450-4679-a8c6-6023c53c6036)

**Comment by AppHock** [comment]
这就问题就是最常见的遍历 Array时，对又对 Array 中的元素进行了增删操作，就看这个NSConcreteHashTable是你们自己项目使用的对象还是 Pag 使用的

**Comment by kevingpqi123** [comment, marked as answer]
你这边使用的是哪个版本，PAG 4.2 和 PAG 4.3 最新版本有修复过类似问题，crash 的堆栈和你这边的不太一致，是 PAGView 中的

**Reply by leiyuncun** [reply]
出现这个问题使用的libpag库的版本：4.1.13，我先更新4.3.33版本观察，谢谢！
